android-mvvm-kotlin/app/build.gradle

```diff
@@ -37,6 +39,18 @@ android {
 }
 
 dependencies {
+    //glide
+    implementation 'com.github.bumptech.glide:glide:4.13.2'
```

Comment:
you should use coin lib intead of glide, becasue glide shoud use for java and coin use for kotlin